### PR TITLE
Feature/248

### PR DESCRIPTION
**Pull request by OksanaSam**
#248
Matching backend structure for Driver Create when sending a POST request to create Driver

### CODE REVIEW

File: web-client/src/shared/components/drivers-registration/PersonalInfo/index.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ const PersonalInfoSection = ({ handleNext, handleBack }: personalInfoProps): Rea
         { name: 'english', label: 'English' },
         { name: 'french', label: 'French' },
         { name: 'tagalog', label: 'Tagalog' },
-        { name: 'portuguese', label: 'Portuguess' },
+        { name: 'portuguese', label: 'Portuguese' },
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/PersonalInfo/index.tsx:73]: Nice catch

File: web-client/src/shared/services/languages.test.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { getCodeFromName, getNameFromCode } from './languages';
```
Comment (l2silver) [inline web-client/src/shared/services/languages.test.ts:1]: Very nice

File: web-client/src/shared/redux/driversRegistration/index.ts [hunk 3]
```diff
@@ -123,10 +127,37 @@ const driversRegistrationSlice = createSlice({
...
+    const convertedRequest = convertRequestToBackendFormat(createDriverRequest);
+    return async (dispatch) => {
+        try {
+
+            const host = process.env.NEXT_PUBLIC_API_SERVER_URL;
+            const response = await fetch(`${host}/drivers`, {
+                method: 'POST',
+                headers: {
+                'Content-Type': 'application/json',
+                body: JSON.stringify({data: convertedRequest})
```
Comment (l2silver) [inline web-client/src/shared/redux/driversRegistration/index.ts:149]: this looks perfect, except for the data property here. Have you been able to create drivers using this method?

File: web-client/src/shared/utils/convertAvailability.test.ts [hunk 4]
```diff
@@ -4,13 +4,17 @@ import { AvailableDate } from "../redux/driversRegistration/yourNeeds/index";
 
 //test cases
```
Comment (l2silver) [inline web-client/src/shared/utils/convertAvailability.test.ts:5]: Very nice

**Review by l2silver — APPROVED** [review]
Wow, great work! This looks excellent.

File: web-client/src/shared/redux/driversRegistration/index.ts (hunk 3 above)
Comment (OksanaSam) [reply web-client/src/shared/redux/driversRegistration/index.ts:149]: no, I haven't. In swagger, there is also a token, not sure where it should come from when the driver is just signing up

Comment (l2silver) [reply web-client/src/shared/redux/driversRegistration/index.ts:149]: Ah cool. I'm gonna push some changes to get that going.

Comment (l2silver) [reply web-client/src/shared/redux/driversRegistration/index.ts:149]: Ah yes, there shouldn't be any token restrictions. I'll have to make the changes in the backend.

**Comment by l2silver** [comment]
Should be fixed now

**Review by l2silver — APPROVED** [review]

**Pull request merged**